src/configs/react.ts

```diff
@@ -72,34 +94,47 @@ export async function react(
           { allowConstantExport: isAllowConstantExport },
         ],
 
-        // recommended rules react
-        'react/display-name': 'error',
-        'react/jsx-key': 'error',
-        'react/jsx-no-comment-textnodes': 'error',
-        'react/jsx-no-duplicate-props': 'error',
-        'react/jsx-no-target-blank': 'error',
-        'react/jsx-no-undef': 'error',
-        'react/jsx-uses-react': 'error',
-        'react/jsx-uses-vars': 'error',
-        'react/no-children-prop': 'error',
-        'react/no-danger-with-children': 'error',
-        'react/no-deprecated': 'error',
+        // recommended rules from @eslint-react
+        'react/ensure-forward-ref-using-ref': 'warn',
+        'react/no-access-state-in-setstate': 'error',
+        'react/no-array-index-key': 'warn',
+        'react/no-children-count': 'warn',
+        'react/no-children-for-each': 'warn',
+        'react/no-children-map': 'warn',
+        'react/no-children-only': 'warn',
+        'react/no-children-prop': 'warn',
+        'react/no-children-to-array': 'warn',
+        'react/no-clone-element': 'warn',
+        'react/no-comment-textnodes': 'warn',
+        'react/no-component-will-mount': 'error',
+        'react/no-component-will-receive-props': 'error',
+        'react/no-component-will-update': 'error',
+        'react/no-create-ref': 'error',
         'react/no-direct-mutation-state': 'error',
-        'react/no-find-dom-node': 'error',
-        'react/no-is-mounted': 'error',
-        'react/no-render-return-value': 'error',
+        'react/no-duplicate-key': 'error',
+        'react/no-implicit-key': 'error',
+        'react/no-missing-key': 'error',
+        'react/no-nested-components': 'warn',
+        'react/no-redundant-should-component-update': 'error',
+        'react/no-set-state-in-component-did-mount': 'warn',
+        'react/no-set-state-in-component-did-update': 'warn',
+        'react/no-set-state-in-component-will-update': 'warn',
         'react/no-string-refs': 'error',
-        'react/no-unescaped-entities': 'error',
-        'react/no-unknown-property': 'error',
-        'react/no-unsafe': 'off',
-        'react/prop-types': 'error',
-        'react/react-in-jsx-scope': 'off',
-        'react/require-render-return': 'error',
+        'react/no-unsafe-component-will-mount': 'warn',
+        'react/no-unsafe-component-will-receive-props': 'warn',
+        'react/no-unsafe-component-will-update': 'warn',
+        'react/no-unstable-context-value': 'error',
+        'react/no-unstable-default-props': 'error',
+        'react/no-unused-class-component-members': 'warn',
+        'react/no-unused-state': 'warn',
+        'react/no-useless-fragment': 'warn',
+        'react/prefer-destructuring-assignment': 'warn',
+        'react/prefer-shorthand-boolean': 'warn',
+        'react/prefer-shorthand-fragment': 'warn',
```

Comment:
So, actually, they are completely separate implementations with unrelated sets of rules? I thought it was a fork

Reply:
It's not just a fork, you can understand their differences from the links provided in the issue.

https://eslint-react.xyz/docs/faq

It does implement some rules in eslint-plugin-react. https://github.com/Rel1cx/eslint-react/issues/85.

For me, its advantages are：

- It's a modern rewrite with TypeScript
- It can lint with type information
- It removes all Stylistic rules